src/main/kotlin/net/lumalyte/lg/interaction/menus/bedrock/BedrockGuildHomeMenu.kt

```
@@ -147,117 +138,37 @@ class BedrockGuildHomeMenu(
-            val targetLocation = Location(
-                world,
-                home.position.x.toDouble() + 0.5,  // Center of block
-                home.position.y.toDouble(),
-                home.position.z.toDouble() + 0.5,  // Center of block
-                player.location.yaw,
-                player.location.pitch
-            )
-
-            // Start countdown timer
-            startTeleportCountdown(targetLocation)
         } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)